<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewHash</class>
 <widget class="QWidget" name="ViewHash">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>673</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>631</width>
         <height>470</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <widget class="control::MwcLabelNormal" name="mwcMQlable_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>320</y>
          <width>311</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Scan blockchain with Viewing Key:</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="walletViewingKey">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>620</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Wallet viewing key</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLabelSmall" name="label_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>621</width>
          <height>101</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:12px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; line-height:138%;&quot;&gt;A viewing key allows others to scan the blockchain and view your wallet's outputs.  Sharing this key makes your MWC wallet’s transactions visible,  compromising the privacy of your funds.&lt;br /&gt;&lt;b&gt;Be cautious when sharing your viewing key - once it’s shared,  your wallet loses its privacy, and all your transactions become transparent to anyone with the key.&lt;/b&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="control::MwcLine" name="frame_4">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>300</width>
          <height>2</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="showWalletKeyButton">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>40</y>
          <width>381</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Show this wallet viewing key</string>
        </property>
        <property name="text">
         <string>Show My Viewing Key</string>
        </property>
       </widget>
       <widget class="control::MwcLineEditNormal" name="viewingKey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>621</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Input viewing key to see the balance</string>
        </property>
        <property name="maxLength">
         <number>1024</number>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="placeholderText">
         <string>Input Viewing Key to scan for outputs</string>
        </property>
       </widget>
       <widget class="control::MwcLabelNormal" name="walletViewingKeyLable">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>My veiwing key:</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="startScanning">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>430</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start skanning for outputs based on viewing key</string>
        </property>
        <property name="text">
         <string>Start scanning</string>
        </property>
       </widget>
       <widget class="control::MwcLine" name="frame_5">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>300</y>
          <width>300</width>
          <height>2</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="generateOwnershipProof">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>240</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start skanning for outputs based on viewing key</string>
        </property>
        <property name="text">
         <string>Generate ownership proof</string>
        </property>
       </widget>
       <widget class="control::MwcPushButtonNormal" name="viewOwnershipProof">
        <property name="geometry">
         <rect>
          <x>340</x>
          <y>240</y>
          <width>221</width>
          <height>40</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="toolTip">
         <string>Start skanning for outputs based on viewing key</string>
        </property>
        <property name="text">
         <string>View ownership proof</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>control::MwcPushButtonNormal</class>
   <extends>QPushButton</extends>
   <header>control_desktop/MwcPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLineEditNormal</class>
   <extends>QLineEdit</extends>
   <header>control_desktop/MwcLineEdit.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelSmall</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
  <customwidget>
   <class>control::MwcLine</class>
   <extends>QFrame</extends>
   <header>control_desktop/MwcFrameWithBorder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>control::MwcLabelNormal</class>
   <extends>QLabel</extends>
   <header>control_desktop/MwcLabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>walletViewingKey</tabstop>
  <tabstop>showWalletKeyButton</tabstop>
  <tabstop>viewingKey</tabstop>
  <tabstop>startScanning</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
